### ISSUE TITLE
OnIndexChange sometimes passes a non-integer number

### ISSUE THREAD

**Issue by Sh3rawi** [react-native-community/react-native-tab-view#385]
### Expected behaviour
OnIndexChange to pass an integer(0, 1, 2)

### Actual behaviour
OnIndexChange sometimes passes a non-integer (0.0013333333333333333, 0.9991111111111112)

### Code sample, screenshots
[https://ibb.co/kkYd2b](url)
```
onIndexChange(index) {
ErrorReport.log('Changes tab to index: ' + index);
const {routes} = this.state;
this.setState({index, currentTab: routes[index].name});
}
```

### What have you tried
I've tried to reproduce this in development but had no luck, also tried to get a pattern out of the fabric data but no luck as well.

**Comment by satya164** [comment]
That's really strange. What platform are you on? Android or iOS?

**Comment by Sh3rawi** [comment]
Its on iOS, ran through a good amount of android crashes and it seems its not happening there.

**Comment by satya164** [comment]
Are you on latest version and using `TabViewPagerScroll`? We always round the value, https://github.com/react-native-community/react-native-tab-view/blob/master/src/TabViewPagerScroll.js#L102

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
